### PR TITLE
`Paywalls`: initial `Template7View` implementation

### PR DESCRIPTION
**Pull request by NachoSoto**
simulator_screenshot_23BBFF48-D097-48FE-ABD2-E3768F243D1B

### CODE REVIEW

File: RevenueCatUI/Templates/Template7View.swift [hunk 1]
```diff
@@ -0,0 +1,467 @@
...
+//
+//  Created by Nacho Soto.
+
+// swiftlint:disable type_body_length file_length
+
+import RevenueCat
+import SwiftUI
+
+@available(iOS 15.0, macOS 12.0, tvOS 15.0, watchOS 8.0, *)
+struct Template7View: TemplateViewType {
```
Comment (NachoSoto) [inline RevenueCatUI/Templates/Template7View.swift:20]: Implementation here.

File: RevenueCatUI/Templates/Template7View.swift [hunk 2]
```diff
@@ -0,0 +1,468 @@
...
+        guard let (firstTier, allTiers, tierNames) = configuration.packages.multiTier else {
+            fatalError("Attempted to display a multi-tier template with invalid data: \(configuration.packages)")
+        }
+
+        self.tiers = allTiers
+        self.tierNames = tierNames
+        self.configuration = configuration
+
+        self._selectedTier = .init(initialValue: firstTier)
+        self._selectedPackage = .init(initialValue: allTiers[firstTier]!.default)
```
Comment (vegaro) [inline RevenueCatUI/Templates/Template7View.swift:64]: Can this be `firstTier.default`?

**Review by vegaro — APPROVED** [review]
Didn't see much to call out. Good job

File: RevenueCatUI/Templates/Template7View.swift (hunk 2 above)
Comment (NachoSoto) [reply RevenueCatUI/Templates/Template7View.swift:64]: `firtTier` is `PaywallData.Tier`. It's `default` is just a package identifier. This way we get the actual `TemplateViewConfiguration.Package`.

**Review by joshdholtz — APPROVED** [review]
:shipit:

**Pull request merged**